src/main/java/pl/javastart/voting/MemberVote.java

```diff
@@ -0,0 +1,17 @@
+package pl.javastart.voting;
+
+public enum MemberVote {
+    ZA("ZA"),
+    PRZECIW("PRZECIW"),
+    WSTRZYMAL_SIE("WSTRZYMAŁ SIĘ");
```

Comment:
Nazwy zmiennych pisz proszę po angielsku, ale "zawartosć" zmiennej może zostać. W tym przypadku poprawne byłoby: `FOR("ZA")`